zerver/webhooks/gitlab/doc.md

```diff
@@ -8,6 +8,11 @@ if you are using this default.
 
 {!git-webhook-url-with-branches.md!}
 
+By default, this integration will create many topics, however by providing
+a `topic` parameter, the integration will only post to one topic, like so:
```

Comment:
Make this `the integration will only post to one topic. You may specify a topic in the webhook URL, like so:`. I think it sounds more structured. :)

Reply:
Yeah, I was wondering if there was a better way to phrase that